packages/config/config/devices/0x0129/ntm625.json

```diff
@@ -0,0 +1,286 @@
+// ASSA ABLOY NTM625
+// Yale nexTouch Sectional Mortise Touch Screen Keypad Lock with cylinder and deadbolt
+{
+	"manufacturer": "ASSA ABLOY",
+	"manufacturerId": "0x0129",
+	"label": "NTM625",
+	"description": "Yale nexTouch Sectional Mortise Touch Screen Keypad Lock with cylinder and deadbolt",
+	"devices": [
+		{
+			"productType": "0x8008",
+			"productId": "0x0a00",
+			"zwaveAllianceId": 2967
+		}
+	],
+	"firmwareVersion": {
+		"min": "1.78",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Silent mode",
```

Comment:
Is there a better name? "Loud" doesn't really fit for "Silent mode" 😅

Reply:
These aren't my names, straight out of the system integrator manual. Something like "Volume" makes a bit more sense, what is policy here do we diverge from the manual?

[nexTouchSectionalMortise_ZWavePlusSystemIntegratorsGuide_NTM.pdf](https://github.com/zwave-js/node-zwave-js/files/5959961/nexTouchSectionalMortise_ZWavePlusSystemIntegratorsGuide_NTM.pdf)

Reply:
So far our policy is to make the configs precise and understandable - manuals are often not. So if you have to diverge, that is encouraged.